connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -224,7 +224,7 @@ object SparkConnectService {
   // different or complex type easily.
   private type SessionCacheKey = (String, String)
 
-  private var server: Server = _
+  private[connect] var server: Server = _
```

Comment:
Why do we need to increase the visibility?

Reply:
It's used here.
https://github.com/apache/spark/blob/ec9b71e2f0e137056951934491883a52a0f22ba8/connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectServer.scala#L31